### PR TITLE
fix: webGL black screenshots

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

...

## How to test the changes?

1. Launch the explorer
2. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### <samp>🤖 Generated by Copilot at 97a311f</samp>

This pull request improves the ScreencaptureCamera feature by using a RenderTexture and the Universal Render Pipeline to capture high-quality screenshots. It also refactors the SkyboxCamera and ScreencaptureCameraBehaviour classes to simplify the code and avoid potential issues. It updates the ScreenshotCamera.asmdef file to add a new assembly reference.

### CODE REVIEW

File: unity-renderer/Assets/DCLFeatures/ScreencaptureCamera/CameraObject/Scripts/ScreenRecorder.cs [hunk 1]
```diff
@@ -20,6 +24,88 @@ public ScreenRecorder(RectTransform canvasRectTransform)
...
+            // int rectX = 100;
+            // int rectY = 100;
+            // int rectWidth = 500;
+            // int rectHeight = 500;
+            //
+            // // Create a Texture2D with the same GraphicsFormat as the RenderTexture and the size of the rect
+            // Texture2D screenshot = new Texture2D(rectWidth, rectHeight, TextureFormat.RGBAHalf, false);
+            //
+            // // Copy the specific rect from the intermediateRenderTexture to the screenshot Texture2D
+            // Graphics.CopyTexture(intermediateRenderTexture, 0, 0, rectX, rectY, rectWidth, rectHeight, screenshot, 0, 0, 0, 0);
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/DCLFeatures/ScreencaptureCamera/CameraObject/Scripts/ScreenRecorder.cs:90]: Can we remove all this commented code?

**Review by sandrade-dcl — APPROVED** [review]
LGTM! (I only left a comment about to remove some commented code)

File: unity-renderer/Assets/DCLFeatures/ScreencaptureCamera/CameraObject/Scripts/ScreenRecorder.cs (hunk 1 above)
Comment (popuz) [reply unity-renderer/Assets/DCLFeatures/ScreencaptureCamera/CameraObject/Scripts/ScreenRecorder.cs:90]: sry @sandrade-dcl , this one should be a draft.... Another one get merged (mentioned in history)

**Review by kwsskulinski — APPROVED** [review]
No black screenshots have been spotted.

**Review by Kinerius — APPROVED** [review]

**Pull request closed without merging**